src/intents.cs

```diff
@@ -11804,6 +12009,49 @@ interface INFlightReservation : NSCopying, NSSecureCoding {
 		INFlight Flight { get; }
 	}
 
+	[Watch (8,0), NoTV, Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
+	[Native]
+	public enum INFocusStatusAuthorizationStatus : long
+	{
+		NotDetermined = 0,
+		Restricted,
+		Denied,
+		Authorized,
+	}
+
+	[Watch (8,0), NoTV, Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface INFocusStatus : NSCopying, NSSecureCoding
+	{
+		[Export ("initWithIsFocused:")]
+		[DesignatedInitializer]
+		IntPtr Constructor ([NullAllowed] NSNumber isFocused);
```

Comment:
You can use [BindAs](https://docs.microsoft.com/en-us/xamarin/cross-platform/macios/binding/binding-types-reference#bindasattribute) here, see swift [docs](https://developer.apple.com/documentation/sirikit/infocusstatus/3821590-init)

```suggestion
		IntPtr Constructor ([NullAllowed] [BindAs (typeof (bool?))] NSNumber isFocused);
```